packages/remix-node/upload/fileUploadHandler.ts

```diff
@@ -88,22 +92,32 @@ export function createFileUploadHandler({
       return;
     }
 
-    let filedir = resolvePath(
+    let dir =
       typeof directory === "string"
         ? directory
-        : directory({ filename, encoding, mimetype })
-    );
+        : directory({ filename, encoding, mimetype });
```

Comment:
The `directory` option should just be a static string. No need for a function here.

Reply:
I disagree, I might want to use the same filename function but store them in different directories if they are images or PDF's for example. This enables that without creating two separate handlers and having to compose them.

Reply:
Actually, we should probably default it to a tmp directory that we create on node. No need to make `directory` a required option here.

Reply:
Good call!